config/redis.yml

```diff
@@ -208,6 +208,9 @@ development: &defaults
   travel_pay_store:
     namespace: travel-pay-store
     each_ttl: 3300 # 55 minutes
+  form_1010_cg_submission_job_claim:
+    namespace: Form1010cg::SubmissionJob
+    each_ttl: 1000000 # persisted for at least 22 sidekiq retries
```

Comment:
Since this lives for so long, I would almost consider persisting to the db. Although, it is ephemeral in nature in the way that it goes away after the submission completes. What about if the job goes to the dead queue? That would be a risk in this case because the key would then expire and no longer be available.

Reply:
Yeah, those are both valid concerns. If a submission job did go to the dead queue and was re-queued, the statsd value that uses this would probably double count how many submissions have had retries. In investigating the redis keys that my team owns that were both set to not expire, it really looks like the implementation should really be reexamined, since neither of them probably should have been using redis for their use cases at all. I can talk to my PM about getting a ticket to look into refactoring this code. I am fine to close this ticket if you'd prefer us to not implement a redis key with a ttl that long, but if we'd prefer to have some expiration on it we can continue on this and still create a ticket to investigate a better way to do it going forward.

Reply:
@LindseySaari Would you prefer we not bother with this current change and instead create a ticket to refactor how we implement this stat to not even use redis? We are going to need to do something similar for the other redis key that was identified, so I figure it would make sense to do those in tandem in a separate ticket. I wasn't sure if the value of at least getting these in the redis-namespace was valuable, or if it's not we can just do a full refactor. I will have to talk to my PM about priority since the scope is a little large than this small change.